packages/design-system/src/components/N8nMarkdown/Markdown.vue

```diff
@@ -154,16 +162,8 @@ export default Vue.extend({
 			return safeHtml;
 		},
 	},
-	data() {
-		return {
-			md: new Markdown(this.options.markdown) // eslint-disable-line @typescript-eslint/no-unsafe-member-access
-				.use(markdownLink, this.options.linkAttributes) // eslint-disable-line @typescript-eslint/no-unsafe-member-access
-				.use(markdownEmoji)
-				.use(markdownTasklists, this.options.tasklists), // eslint-disable-line @typescript-eslint/no-unsafe-member-access
-		};
-	},
 	methods: {
-		onClick(event: MouseEvent) {
+		onClick(event: MouseEvent): void {
```

Comment:
> LGTM! All very minor comments so feel free to ignore.
> 
> > The increase in typecheck errors is expected due to two reasons
> 
> Would it make sense to ignore type errors on spec files temporarily and add `typecheck` to any regular checks we do on `design-system`? I fixed [all the type errors a while ago](https://github.com/n8n-io/n8n/pull/3956) - it'd be nice to prevent them from accumulating again.

Added a task for fixing errors and adding typecheck to CI, but will likely be done after we update testing-library: https://linear.app/n8n/issue/N8N-6318/fix-design-system-typecheck-issues-and-add-typecheck-to-ci